.github/workflows/ci.yml

```diff
@@ -5,7 +5,22 @@ on:
     push:
         branches:
             - master
-    workflow_dispatch:
```

Comment:
This needs to stay

Reply:
Ok. I'm working on it right now, so I will let you know when work is finished and you can proceed review.

Reply:
I am just outlining things that shouldnt have been touched, and you can include them during that process already.